### PR TITLE
Added the cluster_cost to the cro

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [x] enhancement
- [x] documentation
- [x] dependencies

## Description

Adding the cluster details and the cluster_cost.
1. Fetch the cost from the Athena query.
2. Updated the instance_data format. [ Name, instance_id, instance_type, instance_offering, rosa/self ]

### CODE REVIEW

**Comment by kpouget** [comment]
nice, thanks for the work @athiruma 👍🏻

File: jenkins/cloud_resource_orchestration/run_cloud_resource_orchestration.py [hunk 1]
```diff
@@ -21,6 +21,11 @@
 CRO_PORTAL = os.environ['CRO_PORTAL']
 CRO_COST_OVER_USAGE = os.environ['CRO_COST_OVER_USAGE']
 CRO_ES_INDEX = os.environ['CRO_ES_INDEX']
+AWS_ACCESS_KEY_ID_ATHIRUMA_BOT = os.environ['AWS_ACCESS_KEY_ID_ATHIRUMA_BOT']
+AWS_SECRET_ACCESS_KEY_ATHIRUMA_BOT = os.environ['AWS_SECRET_ACCESS_KEY_ATHIRUMA_BOT']
```
Comment (ebattat) [inline jenkins/cloud_resource_orchestration/run_cloud_resource_orchestration.py:25]: AWS_ACCESS_KEY_ID_ATHIRUMA_BOT, AWS_ACCESS_KEY_ID_ATHIRUMA_BOT
Are those the same as:
ATHENA_ACCOUNT_ACCESS_KEY, ATHENA_ACCOUNT_SECRET_KEY

**Review by ebattat — COMMENTED** [review]
Do we need to add any unittest ?

File: jenkins/cloud_resource_orchestration/run_cloud_resource_orchestration.py (hunk 1 above)
Comment (athiruma) [reply jenkins/cloud_resource_orchestration/run_cloud_resource_orchestration.py:25]: Yes, I am using my creds to querying the Athena database.

**Comment by athiruma** [comment]
> Do we need to add any unittest ?

have them already

File: jenkins/cloud_resource_orchestration/run_cloud_resource_orchestration.py (hunk 1 above)
Comment (ebattat) [reply jenkins/cloud_resource_orchestration/run_cloud_resource_orchestration.py:25]: Why 2 different names ?

Comment (athiruma) [reply jenkins/cloud_resource_orchestration/run_cloud_resource_orchestration.py:25]: these are my personal creds, So I named them as my creds.
We need to pass the Athena creds. So I am asking for the Athena creds.

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**